CocoaTypograf/Service/TypografService.swift

```diff
@@ -8,14 +8,26 @@
 
 import Foundation
 
+/// A protocol describing a typograf service.
 public protocol TypografService {
 
+    /// A handler for an processing a result of processed text or a failure.
```

Comment:
Some mess here — "A handler for an processing a result of processed text or a failure" better to be replaced say with "A closure for handling a result of a text processing or a text processing failure".